quarkus/integration-tests/server/src/test/resources/batch.file

```diff
@@ -1 +1 @@
-create cache --template=org.infinispan.DIST_SYNC mybatch
\ No newline at end of file
+create cache --file=target/test-classes/dcache.json mybatch
```

Comment:
is this change required? I added the templates to the REST endpoint. 
https://github.com/infinispan/infinispan/pull/12871/files#diff-6b434edf0f299b314c0f0f6b046e90ca3653460d5e1787d7d26a4a15862a033eR154

Reply:
While adding those template names is ok, I'd prefer it if we didn't rely on them in our tests